rpm/containers-common.spec

```diff
@@ -98,21 +98,29 @@ not required by Skopeo.
 %prep
 %autosetup -Sgit -n %{repo}-%{version_no_tilde}
 
+# Copy manpages to docs subdir in builddir to build before installing.
+cp %{SOURCE1} docs/containers-auth.json.5.md
+cp %{SOURCE2} docs/containers-certs.d.5.md
+cp %{SOURCE3} docs/containers-policy.json.5.md
+cp %{SOURCE4} docs/containers-registries.conf.5.md
+cp %{SOURCE5} docs/containers-registries.conf.d.5.md
+cp %{SOURCE6} docs/containers-registries.d.5.md
+cp %{SOURCE7} docs/containers-signature.5.md
+cp %{SOURCE8} docs/containers-transports.5.md
+cp %{SOURCE9} docs/containers-storage.conf.5.md
```

Comment:
I think this is more duplication than I can tolerate. Can you revert these (lines 102-110 only) back to 'cp XXX docs/` please?

Reply:
will do. I just thought listing the names gave them better visibility, but I'm cool either way.

Reply:
done

Reply:
For the Nasty Three below (shortnames, registries, storage) that have to be dealt with in a special way, yes, it does. For the ones above (lines 102-110), the filename doesn't matter, all we care about is that they are `.md` files that will be processed in a loop. For those, in this block, we don't care about their names. Listing them explicitly forces me and other reviewers to go line by line, comparing each filename against its corresponding `SourceNN` line at the top of the spec, looking for typos misspellings etc. And if any of those ever change, someone will forget to make the corresponding other changes, and life will suck.